6.AUTH_CONVERTER.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE `account`
+		ADD COLUMN `totaltime` INT(10) UNSIGNED NOT NULL DEFAULT '0' AFTER `recruiter`;
```

Comment:
shouldn't this be added in the wiki guide? http://www.azerothcore.org/wiki/TrinityCore-to-AzerothCore-characters-migration

Reply:
https://github.com/azerothcore/wiki/pull/104